src/Files.App/Views/LayoutModes/BaseLayout.cs

```diff
@@ -1371,6 +1362,27 @@ protected async Task ValidateItemNameInputText(TextBox textBox, TextBoxBeforeTex
 			}
 		}
 
+		protected void UpdatePreviewPaneSelection(List<ListedItem>? value)
+		{
+			if (LockPreviewPaneContent)
+				return;
+
+			if (value?.FirstOrDefault() != PreviewPaneViewModel.SelectedItem)
+			{
+				// Update preview pane properties
+				PreviewPaneViewModel.IsItemSelected = value?.Count > 0;
+				PreviewPaneViewModel.SelectedItem = value?.Count == 1 ? value.First() : null;
+
+				// Check if the preview pane is open before updating the model
+				if (PreviewPaneViewModel.IsEnabled)
+				{
+					var isPaneEnabled = ((App.Window.Content as Frame)?.Content as MainPage)?.ShouldPreviewPaneBeActive ?? false;
+					if (isPaneEnabled)
+						PreviewPaneViewModel.UpdateSelectedItemPreview();
```

Comment:
```suggestion
						_ = PreviewPaneViewModel.UpdateSelectedItemPreview();
```
I think it is important to show that this is fire-and-forget.

Reply:
Oh yikes, good catch! I removed it by accident!